CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Add in_app_include option
```

Comment:
```suggestion
- Add `in_app_include` option (#909)
```

Reply:
Even better, I suggest ``Add `in_app_include` option to whitelist paths that should be marked as part of the app (#909)``